Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsForProcessingBusinessService.cs

```diff
@@ -16,4 +17,13 @@ public Task<int> GetUnprocessedTotalCount()
         => this.submissionsForProcessingData
             .GetAllUnprocessed()
             .CountAsync();
+
+    public bool IsSubmissionProcessing(int submissionId)
+    {
+        var result = this.submissionsForProcessingData
+            .GetQuery(s => s.SubmissionId == submissionId)
+            .FirstOrDefault();
+
+        return result != null ? !result.Processing : true;
```

Comment:
Removing negation to improve readability

Reply:
Fixed.